packages/utilities/psammead-styles/colors.js

```diff
@@ -0,0 +1 @@
+export * from './colours';
```

Comment:
I don't think we should need a translation to US English. It can create confusion to be able to use both versions.